Float.Core/UX/Coordinator.cs

```diff
@@ -26,6 +26,11 @@ public abstract class Coordinator : ICoordinator
         /// </summary>
         Page managedPage;
 
+        /// <summary>
+        /// The event args that are pending a finish.
+        /// </summary>
+        EventArgs waitingToFinishEventArgs;
```

Comment:
@EBusch This value needs to be referenced in `HandleNavigation`. They also need to be cleared at some point.